### DISCUSSION TITLE
Error "new row violates row-level security policy for table "objects"

### DISCUSSION THREAD

**Discussion by borngraced** [supabase/supabase#2169] [Questions] [answered]
while going throught the nextjs tutorial in the supabse doc, I got to to a point where i created the upload media widget but  now, when i tried using the uploader to upload a file to supbse storage, I GeT THE ERROR "new row violates row-level security policy for table "objects".

So far, I haven't made any changes to the policy settings on supabse so i wonder where the error is coming from!

what's with error or what do I need to do

**Comment by leopsidom** [comment, marked as answer]
I just had the same problem. I resolved it by creating policies under the storage bucket to allow insert operation for anon users.

**Reply by borngraced** [reply]
thanks! and what does the sql for that looks like please

**Reply by leopsidom** [reply]
I created the policy from one of the existing templates that allow anon users to access public folder (the first one from the template list). It suits my needs.

**Reply by borngraced** [reply]
doesn't wok for me bro

**Reply by borngraced** [reply]
nevermind i got it fixed! Thank you
